File: src/main/java/com/sport/service/web/models/event/EventResponse.java
package com.sport.service.web.models.event;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.Builder;

@Getter
@Builder
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class EventResponse {
    private String name;
    private String description;
    private String placeName;
    private String district;
    private String address;
    private String link;
    private String date;
    private String time;
    private String coordinates;
}

File: src/main/kotlin/com/sport/service/web/models/event/EventResponse.kt
package com.sport.service.web.models.event

data class EventResponse(
    val name: String?,
    val description: String?,
    val placeName: String?,
    val district: String,
    val address: String,
    val link: String?,
    val date: String,
    val time: String?,
    val coordinates: String?
)